ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/extensions/PaywallDataExtensions.kt

```diff
@@ -0,0 +1,99 @@
+package com.revenuecat.purchases.ui.revenuecatui.extensions
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.toArgb
+import androidx.core.os.LocaleListCompat
+import com.revenuecat.purchases.Package
+import com.revenuecat.purchases.paywalls.PaywallColor
+import com.revenuecat.purchases.paywalls.PaywallData
+import java.net.URL
+
+@Composable
+@ReadOnlyComposable
+internal fun PaywallData.Companion.createDefault(packages: List<Package>): PaywallData {
+    return PaywallData.createDefault(packages.map { it.identifier })
+}
+
+@Composable
+@ReadOnlyComposable
+internal fun PaywallData.Companion.createDefault(packageIdentifiers: List<String>): PaywallData {
+    val locale = LocaleListCompat.getDefault()[0].toString()
+
+    return PaywallData(
+        templateName = PaywallData.defaultTemplate,
+        config = PaywallData.Configuration(
+            packages = packageIdentifiers,
+            images = PaywallData.Configuration.Images(
+                background = PaywallData.defaultBackgroundImage,
+                icon = PaywallData.defaultAppIconPlaceholder,
+            ),
+            colors = PaywallData.defaultColors,
+            blurredBackgroundImage = true,
+            displayRestorePurchases = true,
+        ),
+        localization = mapOf(locale to PaywallData.defaultLocalization), // TODO-PAYWALLS: test this
+        assetBaseURL = PaywallData.defaultTemplateBaseURL, // TODO-PAYWALLS: we need to load a resource not a URL
```

Comment:
iOS uses the resource bundle URL, but there's no such thing in Android when loading from resources, so I am unsure of what to do here yet, besides making assetBaseURL a URI, what do you think @tonidero

Reply:
Hmm right... maybe we can wrap the assets around a class that indicates whether it comes from a URL or the bundle and use that while we are processing the `TemplateConfiguration`... Though if `Coil` works with an Uri as well, I'm fine just converting everything to use Uris (I believe we already do right now when creating the assets for the `TemplateConfiguration` anyway)

Reply:
An alternative is to use a placeholder value like https://github.com/RevenueCat/purchases-ios/blob/main/RevenueCatUI/Helpers/PaywallData%2BDefault.swift#L50 so that the template can look up the images differently if that's necessary.